lib/ddtrace/runtime/cgroup.rb

```diff
@@ -0,0 +1,44 @@
+require 'ddtrace/ext/runtime'
+
+module Datadog
+  module Runtime
+    # For control groups
+    module Cgroup
+      LINE_REGEX = /^(\d+):([^:]*):(.+)$/
+
+      Descriptor = Struct.new(
+        :id,
+        :groups,
+        :path,
+        :controllers
+      )
+
+      module_function
+
+      def descriptors(process = 'self')
+        [].tap do |descriptors|
+          begin
+            filepath = "/proc/#{process}/cgroup"
+
+            if File.exist?(filepath)
+              File.open("/proc/#{process}/cgroup").each do |line|
```

Comment:
This should probably be File.open(filepath) so you only have to change it in one place

Reply:
Thanks, must have forgot to update line 24. We'll do this in a future PR.